edmunds.gemspec

```diff
@@ -27,5 +27,5 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'faraday'
   spec.add_dependency 'json'
-  spec.add_dependency 'active_support'
+  # spec.add_dependency 'active_support'
```

Comment:
so this is used in the request.rb class

`require 'active_support/core_ext/hash'`

When I bundle it fails to install but I know I have it... I'm confused.